Client/Frontend/Browser/TabTrayController.swift

```diff
@@ -384,12 +384,21 @@ class TabTrayController: UIViewController {
 
     fileprivate func makeConstraints() {
         collectionView.snp.makeConstraints { make in
-            make.left.bottom.right.equalTo(view)
+            if #available(iOS 11.0, *) {
+                make.left.bottom.right.equalTo(view.safeAreaLayoutGuide)
+            } else {
+                make.left.bottom.right.equalTo(view)
+            }
             make.top.equalTo(self.topLayoutGuide.snp.bottom)
         }
 
         toolbar.snp.makeConstraints { make in
-            make.left.right.bottom.equalTo(view)
+            if #available(iOS 11.0, *) {
+                make.left.right.equalTo(view.safeAreaLayoutGuide)
```

Comment:
I don't think we need to do this for the toolbar. The background of the toolbar should go from edge to edge.
Using `make.left.right.equalTo(view)` here should be okay.